### ISSUE TITLE
Review the design system for focus state - links

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/czi-scientific-python-mgmt#32]
## Acceptance Criteria

- Focus visible criteria - buttons, links, and navigation components meet at least WCAG 2.1 (designs account for changes in [WCAG 2.2](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)
	-  [WCAG 2.1 SC 2.4.7](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible)
- [WCAG SC 1.4.11 - non-text contrast AA](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast) need a minimum 3:1 contrast ratio with adjacent colours
	- [ ] focus styles - links 
	- [ ] focus styles - navbar items
- [WCAG SC 1.4.1 - use of colour](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color)
	- Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element

**Comment by smeragoel** [comment]
So I have been wrangling my brain, trying to come up with a nice design for focus state for links.

I don't have anything concrete now, but I just wanted to put an update here that this is in progress. Everything I'm coming up with seems to be a "bad" solution, but what I think I'll do is just get out all the bad designs and hopefully something will click and I'll have a concrete direction to move in. I might also just post some of those bad designs to get reviews on them if I'm hardstuck. 

I'll be back here (super) soon 😅

**Comment by trallard** [comment]
I think we can go ahead and propose 1,2,3

**Comment by trallard** [comment]
Now that I have more time to sit down and write things, @smeragoel, I think you can open the proposal on the PyData Sphinx repo.

Your last comment with the screenshots + rationale should be good for this - unless you want to add a TLD;R at the top.

**Comment by trallard** [comment]
Seems everyone is in the agreement of using the monochrome version so will mark this as resolved 🎉
